README.md

```diff
@@ -49,7 +49,39 @@ languages and are part of the [SonarQube](http://www.sonarqube.org/), [SonarClou
 
 ## How to contribute
 
-Check out the [contributing](CONTRIBUTING.md) page to see the best places to log issues and start discussions.
+There are many ways you can contribute to the `sonar-dotnet` project, some very easy and others more involved.

```

Comment:
I know this is from the original text, but the sentence: `, some very easy and others more involved.` feels a bit weird to me. Could we get someone to double-check it as we're changing it anyways ?

Reply:
I'll just drop it. 😄 keep it simple